### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by mskapusuz**
### Summary

* Pre-sale question URL added (is redirected to the https://themeisle.com/contact/?utm_source=wordpress.org&utm_medium=product&utm_campaign=about-us)
* Demo URLs of the PPOM Free Inputs were updated.
* New demo URLs added for PPOM Pro features.
 
### Will affect visual aspect of the product

NO

### Test instructions

N/A

### Time
~ 47min

Closes #77 .

### CODE REVIEW

**Comment by mskapusuz** [comment]
@selul could you please review the query params of the pre-sale URL? (https://themeisle.com/contact/?utm_source=wordpress.org&utm_medium=product&utm_campaign=about-us)

**Comment by selul** [comment]
@mskapusuz the referenced issue is wrong, can you adjust it? 

Regarding UTMs you can use the same values/pattern as `tsk_utmify` function, I saw that the free vs pro also has a wrong utm too. 

Also, why is the pre-sales link something that we want to add? I would revert that for now.

File: readme.txt [hunk 1]
```diff
@@ -75,7 +96,7 @@ All inputs are [sanitized and escaped](https://developer.wordpress.org/themes/th
 * **Compatible with all WooCommerce themes.**
 
 
-[See All PRO Features](https://themeisle.com/plugins/ppom-pro/upgrade/?utm_source=themeisle&utm_medium=product&utm_campaign) - [Demos](https://demo-ppom-lite.vertisite.cloud/)
+[See All PRO Features](https://themeisle.com/plugins/ppom-pro/upgrade/?utm_source=themeisle&utm_medium=product&utm_campaign) - [Demos](https://demo-ppom-lite.vertisite.cloud/) - [Ask a Pre-sale Question](https://themeisle.com/contact/?utm_source=wordpress.org&utm_medium=product&utm_campaign=about-us)
```
Comment (selul) [inline readme.txt:99]: Regarding UTMs you can use the same values/pattern as `tsk_utmify` function, I saw that the free vs pro also has a wrong utm too. 

Also, why is the pre-sales link something that we want to add? I would revert that for now.

**Review by selul — CHANGES_REQUESTED** [review]
@mskapusuz the referenced issue is wrong, can you adjust it?

**Review by selul — APPROVED** [review]

**Pull request merged**